src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/ReflectionJsonTypeInfoOfT.cs

```diff
@@ -217,7 +217,7 @@ private void AddPropertiesAndParametersUsingReflection()
             bool isVirtual,
             JsonSerializerOptions options)
         {
-            JsonIgnoreCondition? ignoreCondition = JsonPropertyInfo.GetAttribute<JsonIgnoreAttribute>(memberInfo)?.Condition;
+            JsonIgnoreCondition? ignoreCondition = memberInfo.GetCustomAttribute<JsonIgnoreAttribute>(inherit: false)?.Condition;
```

Comment:
honestly I prefered when we used helper method here so we don't need to do `inherit: false` everywhere

Reply:
To be clear, this is not changing the inherit parameter anywhere, it's simply using the extension method from System.Reflection instead of the handrolled one.